src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
We usually don't test models directly at all - they are considered tested implicitly with the services that use them